src/app/models/vehicle-tech-record.model.ts

```diff
@@ -8,18 +8,14 @@ export interface VehicleTechRecordModel {
   techRecord: TechRecordModel[];
 }
 
-export interface postNewVehicleModel {
-  vin: string;
-  primaryVrm?: string;
-  trailerId?: string;
-  techRecord: TechRecordModel[];
-}
-
-export interface PutVehicleTechRecordModel extends Omit<VehicleTechRecordModel, 'vrms'> {
+export interface PostNewVehicleModel extends Omit<VehicleTechRecordModel, 'vrms' | 'trailerId'> {
```

Comment:
We don't usually sent the partialVin? And we can sen trailerIds for trailers surely?

Reply:
This is now corrected, thanks.